src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -671,4 +671,18 @@ public TeamDetailsBundle getTeamDetailsForStudent(StudentAttributes student) {
         return null;
     }
 
+    private void addInvalidStudentInfo(List<String> invalidityInfo, StudentAttributes student, String sanitizedLine) {
```

Comment:
can we bring these methods up to directly after `getDuplicateIndex` since these methods are related to each other.